src/android/build.gradle

```diff
@@ -1,7 +1,5 @@
-allprojects {
```

Comment:
@exxbrain in my tests the allprojects directive was required.

Reply:
@Proper-Job, thanks! Could you please describe an issue happened without `allprojects`? I removed `allprojects` but probably I was wrong.

Reply:
I'm not exactly sure anymore, but I believe capacitor couldn't resolve the maven repository without the allprojects directive.